Sources/FigmaExport/Loaders/ImagesLoader.swift

```diff
@@ -394,4 +395,20 @@ extension Component {
         
         return false
     }
+    
+    public func useRTL() -> Bool {
+        guard let description = description, !description.isEmpty else { return false }
+        
+        let keywords = ["RTL", "rtl", "Rtl"]
+        let hasNotKeywords = keywords.allSatisfy { !description.contains($0) }
+        if hasNotKeywords { return false }        
+        
+        if (description.contains("RTL") ||
+            description.contains("rtl") ||
+            description.contains("Rtl"))  {
+            return true
+        }
+        
+        return false
```

Comment:
Use `localizedCaseInsensitiveContains` method to perform case-insensitive search.

```suggestion
        return description.localizedCaseInsensitiveContains("rtl")
```